apps/frontend/src/components/global/upload_tabs/splunk/SplunkReader.vue

```diff
@@ -49,6 +49,31 @@
             <v-icon pr-2>mdi-github-circle</v-icon>
             Splunk Configuration
           </v-btn>
+          <br />
+          <span>
+            The Splunk platform accepts any type of data. In particular, it
+            works with all IT streaming
+            <br />
```

Comment:
why are we manually putting breaks in the text?  the component should be able to handle word wrapping responsively

Reply:
So the linter wont bark